### ISSUE TITLE
Proposal: Please start using Semantic Versioning

### ISSUE THREAD

**Issue by KateGo520** [grafana-tools/sdk#103]
I found that this project already [supports Go modules](https://github.com/grafana-tools/sdk/blob/master/go.mod). But sadly, the [tags](https://github.com https://github.com/grafana-tools/sdk/tags) doesn't follow [Semantic Versioning](https://semver.org), which means that all tags of this project will be ignored by Go modules and replaced by pseudo-versions, go get acts weirdly when tags are not in that form. It would be great to have the tagged release be named in the format vX.X.X format so that go mod can read it. 
```
	github.com/grafana-tools/sdk v0.0.0-20200802204627-e6a8cdd8fef5
```
Else the mod file shows something like `github.com/grafana-tools/sdk v0.0.0-20200802204627-e6a8cdd8fef5` which is not very readable and difficult to upgrade. It’s hard to verify which version is in use. This is not conducive to version control.

So, I propose this project to follow Semantic Versioning in future versions. For example, `v1.0.1`, `v2.0.0`, `v3.1.0-alpha`, `v3.1.0-beta.2`etc, so that other project can use tag in go.mod.

**Comment by KateGo520** [comment]
@GiedriusS  @K-Phoen  Could you help me review this issue? Thx :p

**Comment by GiedriusS** [comment]
Yes, we should start doing this starting with `v1.0.0` once https://github.com/grafana-tools/sdk/issues/47 is done.
